packages/cli/README.md

```
@@ -0,0 +1,135 @@
+   curl -fsSL https://raw.githubusercontent.com/AgentWorkforce/relay/main/install.sh | bash
+   ```
+
+2. Install the orchestration skill:
+
+   ```bash
+   npx skills add https://github.com/agentworkforce/skills --skill orchestrating-agent-relay
+   ```
+
+3. Tell your agent to use it:
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)